course05/homework01/freekpaans/string_tests.cc

```diff
@@ -0,0 +1,155 @@
+#include "string.h"
+
+#include <cassert>
+#include <iostream>
+#include <string>
+#include <sstream>
+
+void TestEmpty()
+{
+	String str;
+	assert(str.Empty());
+}
+
+void test_default_ctor_string_has_zero_size()
```

Comment:
I usually prefer snake case for test case names - they're much more readable than camel case for longer sentences, imo.

Reply:
Sure, but we use CamlCase at Optiver